hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -46,6 +47,7 @@
   private long quotaInBytes;
   private long quotaInCounts;
   private final OmOzoneAclMap aclMap;
+  private LongAdder quotaUsageInBytes = new LongAdder();
```

Comment:
QuotaUsageInBytes is a property of Volume that needs to be updated each time when CreateKey, AllocateBlock, CommitKey, DeleteKey, and the lock of Volume is usually used. 
Previously, only Bucket locks were used for these operation, and Volume locks can greatly affect the concurrency performance of different buckets. 
So to avoid Volume locking for performance, LongAdder is used here to complete the atomic update of quotaUsageInBytes.